Gruntfile.js

```diff
@@ -67,6 +68,10 @@ module.exports = function (grunt) {
         if (!version) {
             grunt.fail.fatal('No version given, please provide a target version');
         }
+        let resolveVersion = require('./out/src/launcher/util').resolveRuntimeVersion;
+        resolveVersion(version).then(v => {
```

Comment:
Let's pipe the output of the core build to std out and register building the core as a separate task